frontend/src/routes/docs/rooc/+page.svelte

```diff
@@ -21,10 +21,28 @@
         ROOC documentation
     </h1>
     <p>
-        ROOC is a language designed to parse and convert formal optimization models into static formulations. These
-        static formulations can be transformed into linear models which can then be solved using optimization
-        techniques.
+        ROOC is a language designed to help you formalize problems, and prepares those problems in linear form to be solved. The ROOC environment
```

Comment:
I'm thinking it might be better to not explain what happens "under the hood" in the introduction page as it's not that useful to a first viewer of the website, so maybe I'd change it to something like 

"Rooc is a language designed to help you formalize and solve optimization models" or similar

Reply:
That makes sense. I removed all the implementation details and went with "Rooc is a language designed to help you formalize problems and solve them through optimization models"

I think mentioning straight away that we can formalize problems is a bigger hook. 

If we describe it as "formalize and solve optimisation models" (which is the end goal) it leaves the question "what do you mean by optimisation models?" straight away, whereas everyone knows the struggle or trying to agree on what the problem is.